src/calendar/CalendarCellButton.ts

```diff
@@ -11,7 +11,7 @@ import { createComponent, createHook } from "reakit-system";
 import { ariaAttr, callAllHandlers } from "@chakra-ui/utils";
 import { ButtonHTMLProps, ButtonOptions, useButton } from "reakit";
 
-import { isInvalid } from "./__utils";
+import { isInvalidDate } from "../utils";
```

Comment:
let's change this function from isInvalidDate to isDateInValidRange or something like that 🤔

Reply:
let's fix to `isInvalidDateRange`

Reply:
Yeah ok but we also have something called "DateRangeValue" type which could be confusing with it...

And this function is making sure that the date is in range.. so maybe "isDateInBetween" or "DateIsInRange" something like that

Reply:
IMO they are not confusion since `isInvalidDateRange` it has a keyword `is`.

We can stick it to that.